Library/Homebrew/development_tools.rb

```diff
@@ -78,8 +78,8 @@ def llvm_clang_build_version
     end
 
     sig { params(cc: String).returns(Version) }
-    def non_apple_gcc_version(cc)
```

Comment:
We haven't dealt with an Apple GCC in forever at this point.

Reply:
Agreed. That's a very old naming coming from the gcc 4.2 times if I remember well.